playbooks/experiments/roles/pg_remote_db_populate/tasks/main.yaml

```diff
@@ -0,0 +1,65 @@
+---
+  - name: "Create foreman user"
+    shell:
+            psql -U postgres -c "CREATE USER \"foreman\" WITH PASSWORD '{{ foreman_password }}'"
+    become: True
+    become_user: postgres
+    delegate_to: "{{ pgsql_host }}"
+  - name: "Create candlepin user"
+    shell:
+            psql -U postgres -c "CREATE USER \"candlepin\" WITH PASSWORD '{{ candlepin_password }}'"
+    become: True
+    become_user: postgres
+    delegate_to: "{{ pgsql_host }}"
+  - name: "Create database for Foreman"
+    shell:
+            psql -U postgres -c "CREATE DATABASE foreman OWNER foreman"
+    become: True
+    become_user: postgres
+    delegate_to: "{{ pgsql_host }}"
+  - name: "Create database for candlepin"
+    shell:
+            psql -U postgres -c "CREATE DATABASE candlepin OWNER candlepin"
+    become: True
+    become_user: postgres
+    delegate_to: "{{ pgsql_host }}"
+  - name: "Get the foreman dump from satellite"
```

Comment:
Initially was trying to copy directly to temp, but when tried to login to the server using ssh, the files didn't seem to be there